planning/collision_checker.h

```diff
@@ -751,6 +751,11 @@ class CollisionChecker {
       const int end, Parallelism parallelize = Parallelism::Max()) const;
   //@}
 
+  std::vector<uint8_t> CheckConfigSliceCollisionFree(
```

Comment:
It seems like this method should be called by CheckConfigsCollisionFree, since it's strictly more general.